BritneyS/portfolio-site-gatsby
# Edit intro

## What does this PR do :question:
This PR refactors the intro section into its own component, adds a scroll message, and changes the subheading font to "Helvetica Neue" (with sans-serif fallbacks). Responsive styling was added to the main heading, as well.


## Any packages added :package:
N/A


## How to test it :microscope:
1. Checkout this branch
2. Run the site with `gatsby develop` and navigate to `http://localhost:8000/` (or the URL given in the terminal if you have another service running on that port)
3. Confirm that the styling matches the screenshot provided below
4. Resize the window (or use Responsive mode in Chrome dev tools) and confirm that the main heading ("Hi, I'm Britney!") text doesn't wrap, but shrinks to accommodate the screen size.


## Any background info you would like to include :white_check_mark:
N/A


## Screenshots (if applicable) :camera:
<img width="929" alt="Screen Shot 2020-01-19 at 9 13 42 PM" src="https://user-images.githubusercontent.com/8409475/72693907-eff3ea00-3b00-11ea-8106-0e7593171b02.png">
<img width="751" alt="Screen Shot 2020-01-19 at 9 14 01 PM" src="https://user-images.githubusercontent.com/8409475/72693917-fa15e880-3b00-11ea-9105-d01f41dd532d.png">
<img width="421" alt="Screen Shot 2020-01-19 at 9 14 16 PM" src="https://user-images.githubusercontent.com/8409475/72693920-ff733300-3b00-11ea-901b-0fd2217d3a56.png">
<img width="345" alt="Screen Shot 2020-01-19 at 9 14 40 PM" src="https://user-images.githubusercontent.com/8409475/72693925-039f5080-3b01-11ea-9ef1-a410b2b665a0.png">
